src/components/layout/header.tsx

```
@@ -5,14 +5,31 @@ import { Search, Bell, ShoppingCart } from 'lucide-react';
 import { Button } from '@/components/ui/button';
 import { Input } from '@/components/ui/input';
 import Image from 'next/image';
+import { useUIStore, useAuthStore } from '@/store';
 import { useUIStore } from '@/store';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'useUIStore'. Consider to delete it or rename it.

'useUIStore' is defined here:


(lint/suspicious/noRedeclare)